src/components/atoms/Spinner/Spinner.tsx

```diff
@@ -4,24 +4,15 @@ import { colors } from '../../../constants/colors';
 
 export interface ISpinnerProps {
   /**
-   * Background color (3 quarters of the spinner)
-   */
-  backgroundColor?: string;
-
-  /**
-   * Width of the border
-   */
-  borderWidth?: string;
-
-  /**
-   * Front color (1 quarter of the spinner)
+   * Size of the spinner
+   * @default false
    */
-  frontColor?: string;
-
+  small?: boolean;
```

Comment:
I always prefer `size`, `mode` props with multiple options than a `small`, `negative` toggle props. Even if it has only one option.

Reply:
I see the point, I personally prefer to have these variations on `<Spinner />` to the very minimum ( _hence the `boolean` props to not give room for extension_ ) given how inconsistently it's been used at **Zopa** historically 😓 

TBH I made up this as it wasn't really _spece out_, I'll catch-up with the design team and see what they think.

Reply:
Cool!

Reply:
I too prefer props to have multiple options rather than boolean values, makes it more scalable in the future. Having said this could we not make the spinner fit the container size and have a maxHeight/maxWidth so we can't go larger than that. The spinner is used in a few places e.g buttons, sections, full page.

Reply:
I asked @Gianeo, lets wait his feedback 👍